### ISSUE TITLE
Is the provider "jspm.system" gone now?

### ISSUE THREAD

**Issue by SomethingSexy** [jspm/generator#268]
After updating to 1.1.0, it seems `defaultProvider: 'jspm.system'` is not valid anymore.  Is this still the case? The README still has it in there.

**Comment by guybedford** [comment]
I believe this should now be `defaultProvider: 'jspm.io#system'` due to internal changes. We just need to ensure we update the docs on this.

**Comment by Bubblyworld** [comment]
Yep, this was changed to `jspm.io` and `jspm.io#system` recently differentiate the CDN (`jspm.io`) and the CLI ([`jspm/jspm`](/jspm/jspm)) a little bit. I'll update the docs 👍
